code/modules/vehicle/vehicle.dm

```diff
@@ -146,7 +150,7 @@
 	if(has_gravity(src))
 		return 1
 
-	if(pulledby && pulledby != buckled_mob)	// no pulling the vehicle you're driving through space!
+	if(pulledby && pulledby != buckled_mobs[1])	// no pulling the vehicle you're driving through space!
```

Comment:
gonna try something..and it dosen;t look right to me so may need to scold me later.